src/plugins/share/public/components/url_panel_content.tsx

```diff
@@ -100,8 +117,8 @@ export class UrlPanelContent extends Component<Props, State> {
       <I18nProvider>
         <EuiForm className="kbnShareContextMenu__finalPanel" data-test-subj="shareUrlForm">
           {this.renderExportAsRadioGroup()}
-
           {this.renderShortUrlSwitch()}
+          {this.renderUrlParamExtensions()}
```

Comment:
This is always rendered even when the embedding application doesn't act on the parameters - e.g. Visualize

If we are not moving this completely into the dashboard plugin, we should at least put it behind a config flag.